opennlp-tools/src/main/java/opennlp/tools/namefind/RegexNameFinder.java

```diff
@@ -178,18 +197,34 @@ public void clearAdaptiveData() {
     // nothing to clear
   }
 
+  /**
+   * @return Retrieves the {@link Pattern patterns} used.
+   */
+  // TODO Unclear what 'm' stands for: Discuss refactoring, deprecation, or removal of unused method.
   public Pattern[] getmPatterns() {
```

Comment:
Matches? Or a typo...

Reply:
I believe it refers to the pattern (java.util.regex.Pattern.class) of the matched value. (ref : RegexNameFinder.java -> L92)

Reply:
@kinow @atarora What's your opinion in terms of 
"refactoring, deprecation, or removal" of this unused method?

Either
- (A)  refactor it to `getMatchingPatterns()`
- (B)  refactor it to `getMatchingPatterns()` and deprecate
- (C) leave method name unchanged and provide an explanation of "m" via JavaDoc text
- (D)  remove without deprecation period

Reply:
+1 to A or B

Reply:
Please @rzo1 share your thoughts on (A) - (D) and leave a thumb or separate comment.

Reply:
A or B

Reply:
Option A it is, will proceed with that.